go.mod

```diff
@@ -261,24 +261,24 @@ require (
 	github.com/spf13/cast v1.7.1 // indirect
 	github.com/subosito/gotenv v1.6.0 // indirect
 	github.com/vbatts/tar-split v0.11.6 // indirect
-	go.mondoo.com/mondoo-go v0.0.0-20250326000352-c1087ab878cb
+	go.mondoo.com/mondoo-go v0.0.0-20250423000358-4dfe1236d293
 	go.opentelemetry.io/otel/trace v1.35.0 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
-	golang.org/x/exp v0.0.0-20250305212735-054e65f0b394
+	golang.org/x/exp v0.0.0-20250408133849-7e4ce0ab07d0
 	golang.org/x/mod v0.24.0
-	golang.org/x/oauth2 v0.28.0 // indirect
+	golang.org/x/oauth2 v0.29.0 // indirect
 	golang.org/x/time v0.11.0 // indirect
 	golang.org/x/xerrors v0.0.0-20231012003039-104605ab7028 // indirect
-	google.golang.org/api v0.228.0 // indirect
-	google.golang.org/genproto/googleapis/api v0.0.0-20250324211829-b45e905df463 // indirect
+	google.golang.org/api v0.230.0 // indirect
+	google.golang.org/genproto/googleapis/api v0.0.0-20250425173222-7b384671a197 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.28.9 // indirect
 	k8s.io/klog/v2 v2.120.1 // indirect
-	modernc.org/libc v1.62.1 // indirect
+	modernc.org/libc v1.64.0 // indirect
 	modernc.org/mathutil v1.7.1 // indirect
-	modernc.org/memory v1.9.1 // indirect
-	modernc.org/sqlite v1.35.0 // indirect
+	modernc.org/memory v1.10.0 // indirect
+	modernc.org/sqlite v1.37.0 // indirect
```

Comment:
I think things went wrong last time we bumped this sqlite dep up https://github.com/mondoohq/cnquery/pull/5433/files

Reply:
Seems to be working:
```
--> [os] generate CLI json
--> looking for plugin in .
--> writing plugin json to dist/os.json
--> [os] creating providers/os/dist/os
```

When we see this fail somewhere, we need to revert and also pin, so that the automatic update does not bump it again.

Reply:
Seems, to only fail on x86: https://github.com/mondoohq/cnquery/actions/runs/14859594019/job/41720945297